root/components/home/Explore.js

```diff
@@ -0,0 +1,172 @@
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+const Explore = (): React.Element<'section'> => (
+  <section className="bs" id="explore">
+    <div className="container pt-4 pb-4">
+      <div className="text-center pt-4">
+        <h2 className="mb-2 pb-2 text-uppercase fs-1 fw-bold">
+          {l('Explore MusicBrainz')}
+        </h2>
+      </div>
+      <div className="row">
+        <div className="col-md-6">
+          <div className="card p-4 m-4 shadow">
+            <div className="card-body">
+              <h5 className="text-center mb-4 fs-3 fw-bold">
+                <a
+                  href="https://community.metabrainz.org"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Community')}
+                </a>
+              </h5>
+              <p className="card-text fs-4">
+                {l(`
+                  The forums are where our global community of users 
+                  meet for discussions about the project. 
+                  We invite you to join if you have any questions or comments!
+                `)}
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className="col-md-6 d-flex">
+          <div className="card p-4 m-4 shadow">
+            <div className="card-body">
+              <h5 className="text-center mb-4 fs-3 fw-bold">
+                <a
+                  href="/doc/Development"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Development')}
+                </a>
+              </h5>
+              <p className="card-text fs-4">
+                {l(`
+                  Our developer resources will help you make
+                  use of our data or contribute code. If you are a
+                  commercial user, our live data feed will keep
+                  your local database in sync.
+                `)}
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className="col-md-6 d-flex">
+          <div className="card p-4 m-4 shadow">
+            <div className="card-body">
+              <h5 className="text-center mb-4 fs-3 fw-bold">
+                <a
+                  href="/doc/About/History"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('History')}
+                </a>
+              </h5>
+              <p className="card-text fs-4">
+                {l(`
+                  Robert Kaye founded MusicBrainz as a
+                  free open music database. The project has since grown
+                  rapidly from a one-man operation to an international
+                  community of enthusiasts that appreciate
+                  both music and music metadata.
+                `)}
```

Comment:
@mayhem Please go through the latest conversations once and let me know if you would request some changes in the wordings :)

Reply:
"MusicBrainz was founded in 2000 as a free open music database. The project has since grown to an international community of enthusiasts that appreciate both music and music metadata."
@reosarevok does this sound fine?

Reply:
Maybe "MusicBrainz was founded in 2000 as a free open music database. The project has since grown from a basic database for CDs to a large, deep source of all sorts of music information maintained by an international community of enthusiasts that appreciate both music and music metadata."? But I'm fine with the original too, really, if Rob is.

Reply:
Done!